build.gradle

```diff
@@ -221,6 +221,9 @@ subprojects {
                 url "${localNexusLocation}/public"
             }
         } else {
+            maven {
```

Comment:
I think it would be better to place this part common for both if else branches before if statement and negate the condition

Reply:
will be removed once proper xgb is released, before merging this